src/core/probe/kernel/kernel.rs

```diff
@@ -36,6 +38,20 @@ impl KernelProbe {
             config: desc.probe_cfg,
         })
     }
+
+    fn fixup_config(&mut self, opt: &ProbeOptions) {
+        match opt {
+            ProbeOptions::StackTrace => {
+                self.config.stack_trace = 1;
+            }
+        }
+    }
```

Comment:
I split that with the idea of making it easy to turn it visible to other users as soon as a non-local one required it. Make sense to export this rather than the iterating one.